private/openapi-3-v2.0.yaml

```diff
@@ -0,0 +1,212 @@
+openapi: 3.0.0
```

Comment:
why we use yaml as we are using jsons so far( also for public) ?

Reply:
As I was struggling with editing the spec in JSON, I thought "well, we might as well switch to YAML since it's less painful". I didn't remember that we need to have the spec in JSON format because we're serving it on an endpoint, and for the sake of consistency, I'm transforming the private spec back too to JSON.